libcontainer/cgroups/utils.go

```diff
@@ -257,7 +257,13 @@ func RemovePath(path string) error {
 	}
 
 	infos, err := os.ReadDir(path)
-	if err != nil && !os.IsNotExist(err) {
+	if err != nil {
+		if os.IsNotExist(err) {
+			// Please keep this error eraser, or else it will return ErrNotExist
+			// for cgroupv2.
+			// Please see https://github.com/opencontainers/runc/issues/4518
```

Comment:
Would it be possible to have a bats test?

Reply:
I think it’s hard to simulate the similar error.
But maybe we can split this func, and add a go unit test. I don’t know it’s worth to do or not. But we should verify this patch is correct or not at first.